.github/workflows/linelint.yml

```diff
@@ -0,0 +1,13 @@
+# .github/workflows/main.yml
```

Comment:
This line is redundant. Since `linelint.yml` is added, `lint.yml` should be removed.